include/aikido/util/KinBodyParser.hpp

```diff
@@ -0,0 +1,61 @@
+#ifndef AIKIDO_UTIL_KINBODYPARSER_HPP_
+#define AIKIDO_UTIL_KINBODYPARSER_HPP_
+
+#include <dart/dart.hpp>
+
+namespace aikido {
+namespace util {
+
+/// Read skeleton from a string of OpenRAVE's custom XML format
+///
+/// This function only parses a subset of the format assuming only one body node
+/// in a kinbody file.
+///
+/// Followings are the available fields that this parser can read.
+/// \code
+/// kinbody - attributes: name
+///   body - attributes: name
+///     geom - attributes: name, type* (none, box, sphere, trimesh, cylinder), render
+///       data* (or collision) - file* scale (e.g., /path/to/mesh/file_collision.stl 0.5) [for trimesh]
+///       extents* - 3 float [for box]
+///       height* - float [for cylinder]
+///       radius* - float [for cylinder and sphere]
+///       render - file* scale (e.g., /path/to/mesh/file_render.stl 0.5)
```

Comment:
See my note above about `scale`. The same applies here (see [this snippet](https://github.com/rdiankov/openrave/blob/18521d4ee6311464522a4e98cf0eed883a2d15d3/src/libopenrave/xmlreaders.cpp#L400)).